.github/workflows/scripts/linux_arm64/build_enzyme.sh

```diff
@@ -0,0 +1,30 @@
+set -e -x
+cd /catalyst 
+# Process args
+export GCC_VERSION=$1
+export PYTHON_VERSION=$2
+export PYTHON_SUBVERSION=$3
+export PYTHON_PACKAGE_NAME=$4
+# Install system dependencies
+dnf update -y 
+dnf install -y libzstd-devel gcc-toolset-${GCC_VERSION} ${PYTHON_PACKAGE_NAME} ${PYTHON_PACKAGE_NAME}-devel
+dnf clean all -y 
+# Make GCC the default compiler
+source /opt/rh/gcc-toolset-${GCC_VERSION}/enable -y 
+export C_COMPILER=/opt/rh/gcc-toolset-${GCC_VERSION}/root/usr/bin/gcc 
+export CXX_COMPILER=/opt/rh/gcc-toolset-${GCC_VERSION}/root/usr/bin/g++ 
+# Set the right Python interpreter
+rm -rf /usr/bin/python3
+ln -s /opt/_internal/cpython-${PYTHON_VERSION}.${PYTHON_SUBVERSION}/bin/python3 /usr/bin/python3 
+export PYTHON=/usr/bin/python3
+# Add LLVM, Python and GCC to the PATH env var
+export PATH=/catalyst/llvm-build/bin:/opt/_internal/cpython-${PYTHON_VERSION}.${PYTHON_SUBVERSION}/bin:/opt/rh/gcc-toolset-${GCC_VERSION}/root/usr/bin:$PATH
+# Install python dependencies
+/usr/bin/python3 -m pip install numpy pybind11 PyYAML cmake ninja
```

Comment:
To prevent issues with the upcoming numpy 2.0 release, I'd suggest version pinning numpy here to the same as PennyLane (or installing PL as a pre-requisite throughout)

Reply:
If I go for the latter, should I not install numpy? Or should I also keep it?

Reply:
You probably can install pennylane without asking for numpy, as PL will bring in the required version it needs (and we have <2.0 pinned in there already, so the deps carry across here).

Reply:
Done